packages/gatsby-plugin-react-css-modules/src/gatsby-node.js

```diff
@@ -1,8 +1,16 @@
 exports.onCreateBabelConfig = ({ actions }, pluginOptions) => {
+  const isDevelopment = process.env.NODE_ENV !== `production`
+  const longNames = `[name]--[local]--[hash:base64:5]`
+  const shortNames = `[hash:base64:5]`
+  const minifyClassNames = pluginOptions.minifyClassNames || false
+
+  delete pluginOptions.minifyClassNames
+  //Let's delete minifyClassNames option as it is not needed by babel-plugin-react-css-modules
+
   actions.setBabelPlugin({
     name: `babel-plugin-react-css-modules`,
     options: {
-      generateScopedName: `[name]--[local]--[hash:base64:5]`,
+      generateScopedName: isDevelopment ? longNames : (minifyClassNames ? shortNames : longNames) ,
```

Comment:
is there any reason not to minify class names in production? we could probably get away without extra setting and just do it - and people can still overwrite it if they want in pluginOptions by passing `generateScopedName` there

Reply:
Good point. Will update that